.tools/readmes/render.py

```diff
@@ -331,7 +332,8 @@ def write(self):
                 self.readme_filename,
                 f'{self.lang_config["service_folder"]}/{config.saved_readme}',
             )
-        os.remove(self.readme_filename) #  Do this so that new files are always updated to the correct case (README.md).
+        # Do this so that new files are always updated to the correct case (README.md).
+        Path(self.readme_filename).unlink(missing_ok=False)
```

Comment:
I think we wanted `missing_ok=True`, right? The bug was that missing files were throwing an error on delete?